packages/remix/test/fixtures/13-remix-v2-public-path/vercel.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Delete?

Reply:
If this file is deleted, then will the build find the `rootDirectory`?